### DISCUSSION TITLE
Feature Request: UMD Bundling & SRI

### DISCUSSION THREAD

**Discussion by fulldecent** [wevm/viem#1143] [Idea / Feature Request]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

1.9.5

### Current Behavior

Loading the simple way (just include from CDN) does not support SRI.

### Expected Behavior

Support SRI.

### Steps To Reproduce

See https://www.jsdelivr.com/package/npm/viem

There is no file that works in your browser that does SRI.

**Comment by jxom** [comment]
You can get the SRI from unpkg: https://unpkg.com/viem@1.9.5/dist/cjs/index.js?meta

But if you mean it’s not working at all, I think that is because we:
1. Don’t bundle as UMD
2. SRI doesn’t work with ESM.

**Comment by fulldecent** [comment]
Right, I am sorry for the miswording. My issue here is that Viem does not bundle as UMD.

Therefore anybody that uses CDN (as ESM) will gain a supply chain security risk. (i.e. the CDN can change files to anybody, even just for specific individuals, without being detected.)

I request, please, that this issue remain open and consideration be made for publishing as UMD, since supply chain security is a best practice.
